client/state/posts/utils.js

```diff
@@ -208,3 +208,17 @@ export function getTermIdsFromEdits( post ) {
 		} )
 	};
 }
+
+/**
+ * Returns a normalized post object for sending to the API
+ *
+ * @param  {Object} post Raw post object
+ * @return {Object}      Normalized post object
+ */
+export function normalizePostForAPI( post ) {
```

Comment:
Camel-case applies to acronyms: `normalizePostForApi` ([guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions))